src/main/java/teammates/ui/automated/StudentCourseJoinEmailWorkerAction.java

```diff
@@ -43,7 +44,7 @@ public void execute() {
         try {
             emailSender.sendEmail(email);
         } catch (Exception e) {
-            throw new RuntimeException("Unexpected error while sending email", e);
+            Assumption.fail("Unexpected error while sending email" + TeammatesException.toStringWithStackTrace(e));
```

Comment:
Will this affect the exception stack? If an exception happens in the LIVE server, can we trace down to the root cause?

Reply:
Is there anyway I can check this?

Assumption.fail() eventually throws `AssertionError`, so the exception stack will still be useful in tracing down the root cause. [ If there is a way to check, please let me know so that I can confirm ]